### ISSUE TITLE
Setup my lab environment => Exercise 1 => Account Info Screen

### ISSUE THREAD

**Issue by VitalyMCT** [MicrosoftLearning/PL-500T00-Microsoft-Power-Automate-RPA-Developer#95]
At the end of this exercise students get the following screen:

![image](https://user-images.githubusercontent.com/1765732/190035447-16223394-3de9-43db-a4d9-26cb3958f8d6.png)

How to proceed? Can the instructions please be amended to provide clarify (i.e. do they need to enter this info, ignore it, etc.)?

**Comment by VitalyMCT** [comment]
@davidyack Can you please amend the instructions to cover the current sign-up workflow?

**Comment by julians1966** [comment]
Entering 0123456789 is sufficient for the phone number.

**Comment by VitalyMCT** [comment]
@skulthm-ms Have you had a chance to investigate this item?

This gap in instructions is creating a range of issues for attendees and MCTs. 

Example from session today, one of my attendees wanted to create the environment in her locale and got this error:

![image](https://user-images.githubusercontent.com/1765732/196280942-5d897aa7-4f56-4c64-a168-24b92659986e.png)

This is all within the supported ALH (Skillable). We really need to bring this aspect of the instructions up to date with actual Power Platform behavior to position attendees towards success with the labs. We cannot leave them to guess and/or try various permutations to find the ones that work.

**Comment by skulthm-ms** [comment]
@VitalyMCT Since you have a class right now. I would recommend that you enter "0123456789" or "0000000000" as phone number and "unitedstates" as locale/region. This is strictly when using an account from the ALH, NOT for personal trial accounts. We are looking into way to suppress the dialog box within the supported ALH.

**Comment by VitalyMCT** [comment]
@skulthm-ms My class has long since passed this setup step, so any fixes would be much too late for it. We already had to develop our own fixes to get the attendees to complete the associated steps.

I am investing a lot of time to file the issues in this repo in the hope they would be investigated fully and have updates made to instructions as a result. I am hoping this would start to happen on a cadence that's of a reasonably prompt frequency.

This particular issue was filed over 1 month ago. There have not been any updates to instructions around it during all that time. I think there is a huge opportunity here to make improvements.

There is a better way to get attendees into the labs than giving them the official instructions + a long list of errata / handwritten workarounds / explanations of when to follow instructions and when to do something else / misc patches. The attendees should be able to simply rely on the official instructions, as written, to succeed with the PL-500 labs.

**Comment by MOC-Labs-Review** [comment]
MOC Labs Review confirms this issue.

**Comment by ephiax20** [comment]
Updated instructions to include adding the dummy phone number 0123456789 to get past the screen.

**Comment by VitalyMCT** [comment]
@ephiax20 

Regarding

> Updated instructions to include adding the dummy phone number 0123456789 to get past the screen.

This only partially addresses the issue described in this ticket. We need to provide _complete guidance_ to the attendees so they can be successful with the labs.

Email address: what should they be entering? The email address from the auto-assigned M365 tenant? Something else?

Country region: what should they be entering? Keep at default, which may be US? Use their locale (this, per the ticket, seems problematic, at least in some scenarios)? Something else?

**Comment by ephiax20** [comment]
During my testing, the email address was already populated by default.  It is normal in the instructions to only mention items which need to be changed and accept other defaults- but I can specifically add guidance to leave everything else as default if you think that's required??

**Comment by VitalyMCT** [comment]
@ephiax20 Yes, I think that would work. Any method of clarifying the steps that can be provided so the attendees can be successful is a valid solution IMO. Has someone validated whether US is the default across ALHs / ALH VM regions, etc.?
